spec/unit/lib/temporal_spec.rb

```diff
@@ -1,16 +1,17 @@
 require 'temporal'
 require 'temporal/workflow'
+require 'pry'
 
 describe Temporal do
   describe 'client operations' do
     let(:client) { instance_double(Temporal::Client::GRPCClient) }
 
     before { allow(Temporal::Client).to receive(:generate).and_return(client) }
-    after { described_class.remove_instance_variable(:@client) }
+    after { described_class.remove_instance_variable(:@client) rescue NameError }
```

Comment:
Hmm… why `rescue NameError`?

Reply:
This is because some specs don't actually generate a client because they stub a response which prevents `generate` ever being called. In these cases removing the instance variable throws a NameError because it doesn't exist. I can either rescue NameError or I can check if the ivar is present before trying to remove, no real preference. I did this because it's shorter.

Reply:
Makes sense 👍